sdk/communication/communication-rooms/karma.conf.js

```diff
@@ -48,10 +48,10 @@ module.exports = function (config) {
     // https://www.npmjs.com/package/karma-env-preprocessor
     envPreprocessor: [
       "TEST_MODE",
-      "COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING",
+      "COMMUNICATION_CONNECTION_STRING_ROOMS",
+      "COMMUNICATION_SAMPLES_CONNECTION_STRING",
```

Comment:
Un like other language SDKs, why do we need different string for sample and one for live test?
Also the name COMMUNICATION_SAMPLES_CONNECTION_STRING is quite odd to me. COMMUNICATION_SAMPLETEST_CONNECTION_STRING_ROOMS   (just like live test)?  
COMMUNICATION_LIVETEST_CONNECTION_STRING_ROOMS

Reply:
In JS samples are generated for JS and TS along with a generated README files. So the pattern is to use a samples connection string with is common in all sample readme in Communication services.

Reply:
Its the same name used in other services and same is initialized in keyvaults as well